assets/wizards/readerRevenue/views/emails/index.js

```diff
@@ -14,41 +14,69 @@ import { values } from 'lodash';
 /**
  * Internal dependencies
  */
-import { PluginInstaller, Settings, withWizardScreen } from '../../../../components/src';
+import { PluginInstaller, ActionCard, Notice, withWizardScreen } from '../../../../components/src';
 
-const { SettingsCard } = Settings;
-
-const SECTIONS = values( newspack_reader_revenue.emails );
+const EMAILS = values( newspack_reader_revenue.emails );
 
 const Emails = () => {
 	const [ pluginsReady, setPluginsReady ] = useState( null );
-	return pluginsReady === true ? (
-		SECTIONS.map( ( section, i ) => (
-			<SettingsCard key={ i } title={ section.label } columns={ 1 }>
-				<div>
-					<p>{ section.description }</p>
-					{ section.post_id === 0 ? (
-						<div>{ __( 'Missing email', 'newspack' ) }</div>
-					) : (
-						<a href={ section.edit_link }>
-							{ sprintf(
-								/* translators: %s: Email subject */
-								__( 'Edit the "%s" email', 'newspack' ),
-								section.subject
-							) }
-						</a>
+
+	if ( ! pluginsReady ) {
+		return (
+			<>
+				<Notice isError>
+					{ __(
+						'Newspack uses Newspack Newsletters to handle editing email-type content. Please activate this plugin to proceed.',
+						'newspack'
 					) }
-				</div>
-			</SettingsCard>
-		) )
-	) : (
-		<PluginInstaller
-			style={ pluginsReady ? { display: 'none' } : {} }
-			plugins={ [ 'newspack-newsletters' ] }
-			onStatus={ res => setPluginsReady( res.complete ) }
-			onInstalled={ () => window.location.reload() }
-			withoutFooterButton={ true }
-		/>
+				</Notice>
+				<Notice isError>
+					{ __(
+						'Until this feature is configured, default Stripe receipts will be used.',
+						'newspack'
+					) }
+				</Notice>
+				<PluginInstaller
+					style={ pluginsReady ? { display: 'none' } : {} }
+					plugins={ [ 'newspack-newsletters' ] }
+					onStatus={ res => setPluginsReady( res.complete ) }
+					onInstalled={ () => window.location.reload() }
+					withoutFooterButton={ true }
+				/>
+			</>
+		);
+	}
+
+	return (
+		<>
+			{ EMAILS.map( email => {
+				const isActive = email.status === 'publish';
+				return (
+					<ActionCard
```

Comment:
Non-blocking: what if we added a toggle switch to activate (publish) or deactivate (unpublish) the email, similar to how we treat prompts in the Campaigns wizard?

Reply:
I think it's more distraction than usefulness – these emails are rather meant to be "always on" and maybe updated from time to time.